src/Sylius/Bundle/CoreBundle/spec/CommandDispatcher/BatchedApplyCatalogPromotionsOnVariantsCommandDispatcherSpec.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\CoreBundle\CommandDispatcher;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\CoreBundle\Command\ApplyCatalogPromotionsOnVariants;
+use Sylius\Bundle\CoreBundle\CommandDispatcher\ApplyCatalogPromotionsOnVariantsCommandDispatcherInterface;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\MessageBusInterface;
+
+final class BatchedApplyCatalogPromotionsOnVariantsCommandDispatcherSpec extends ObjectBehavior
+{
+    public function let(MessageBusInterface $messageBus,): void
+    {
+        $this->beConstructedWith($messageBus, 2);
+    }
+
+    function it_implements_its_interface(): void
```

Comment:
`it_implements_apply_catalog_promotions_on_variants_command_dispatcher_interface` 🔥 I know it's long, but spec steps should be precise

Reply:
My favorit:
```suggestion
    function it_is_apply_catalog_promotions_on_variants_command_dispatcher(): void
```